docs/data_access.rst

```diff
@@ -129,42 +145,22 @@ HTTPS using the following links:
    be quite large when uncompressed. To decompress them locally, you can use the
    ``gunzip`` command.
 
-
    .. code-block:: console
 
       $ gunzip *.sqlite.gz
 
-
 .. _access-zenodo:
 
 ---------------------------------------------------------------------------------------
-Zenodo Archives
+Zenodo
 ---------------------------------------------------------------------------------------
 
-We use Zenodo to archive our fully processed data as SQLite databases and
-Parquet files. We also archive a Docker image that contains the software environment
-required to use PUDL within Jupyter Notebooks. You can find all our archived data
-products in `the Catalyst Cooperative Community on Zenodo
-<https://zenodo.org/communities/catalyst-cooperative/>`__.
-
-* The current version of the archived data and Docker container can be
-  downloaded from `This Zenodo archive <https://doi.org/10.5281/zenodo.3653158>`__
-* Detailed instructions on how to access the archived PUDL data using a Docker
-  container can be found in our `PUDL Examples repository
-  <https://github.com/catalyst-cooperative/pudl-examples/>`__.
-* The SQLite databases and Parquet files containing the PUDL data, the complete FERC 1
-  database, and EPA CEMS hourly data are contained in that same archive, if you want
-  to access them directly without using PUDL.
-
-.. note::
-
-   If you're already familiar with Docker, you can also pull
-   `the image we use <https://hub.docker.com/r/catalystcoop/pudl-jupyter>`__ to run
-   Jupyter directly:
-
-   .. code-block:: console
-
-      $ docker pull catalystcoop/pudl-jupyter:latest
+We use Zenodo to archive and version our raw data inputs, the fully processed outputs,
+and the PUDL software repositories. You can find all of our archives in
+`the Catalyst Cooperative Community <https://zenodo.org/communities/catalyst-cooperative/>`__.
+Zenodo assigns long-lived DOIs to each archive, suitable for citation in academic
+journals and other publications. The most recent versioned PUDL data release can be
+found using this Concept DOI: https://doi.org/10.5281/zenodo.3653158
```

Comment:
1000% nit but using the "Concept DOI" language is jargony and imo not adding anything.